grpc/src/test/java/com/linecorp/armeria/it/grpc/GrpcDecoratingServiceTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.it.grpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linecorp.armeria.client.grpc.GrpcClients;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.grpc.GrpcSerializationFormats;
+import com.linecorp.armeria.grpc.testing.Messages.SimpleRequest;
+import com.linecorp.armeria.grpc.testing.Messages.SimpleResponse;
+import com.linecorp.armeria.grpc.testing.TestServiceGrpc.TestServiceBlockingStub;
+import com.linecorp.armeria.grpc.testing.TestServiceGrpc.TestServiceImplBase;
+import com.linecorp.armeria.server.DecoratingHttpServiceFunction;
+import com.linecorp.armeria.server.HttpService;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.annotation.Decorator;
+import com.linecorp.armeria.server.grpc.GrpcService;
+import com.linecorp.armeria.server.logging.LoggingService;
+import com.linecorp.armeria.testing.junit5.server.ServerExtension;
+
+import io.grpc.stub.StreamObserver;
+
+public class GrpcDecoratingServiceTest {
+
+    @RegisterExtension
+    static ServerExtension server = new ServerExtension(true) {
+        @Override
+        protected void configure(ServerBuilder sb) {
+            sb.requestTimeoutMillis(5000);
+            sb.decorator(LoggingService.newDecorator());
+            sb.service(GrpcService.builder()
+                                  .addService(new TestServiceImpl())
+                                  .build());
+        }
+    };
+
+    private static String FIRST_TEST_RESULT = "";
+
+    @Test
+    void methodDecorators() {
+        final TestServiceBlockingStub client = GrpcClients.builder(server.httpUri(
+                                                                  GrpcSerializationFormats.PROTO))
+                                                          .build(TestServiceBlockingStub.class);
```

Comment:
We don't need to specify `GrpcSerializationFormats.PROTO` to `GrpcClients`. No `GrpcSerializationFormats` is set, `GrpcSerializationFormats.PROTO` is used by default.
```suggestion
        final TestServiceBlockingStub client = 
                GrpcClients.newClient(server.httpUri(), TestServiceBlockingStub.class);
```